ml-agents/mlagents/trainers/training_status.py

```diff
@@ -103,6 +105,14 @@ def set_parameter_state(category: str, key: StatusType, value: Any) -> None:
         """
         GlobalTrainingStatus.saved_state[category][key.value] = value
 
+    @staticmethod
+    def update_parameter_state(new_saved_state: Dict[str, Dict[str, Any]]) -> None:
```

Comment:
This method does not seem to be used. If we think it will be a useful method for later, we should add a unit test

Reply:
Good call -- I agree it shouldn't be here, will remove